docs/releases/5.0.txt

```diff
@@ -315,6 +315,8 @@ Models
   :ref:`Choices classes <field-choices-enum-types>` directly instead of
   requiring expansion with the ``choices`` attribute.
 
+* :attr:`.Field.choices` now accepts dictionary values as well as sequences.
```

Comment:
shall we add a similar note in the Forms section? Also perhaps merge this entry with the preceding one?